props/saber_BC_buttons.h

```diff
@@ -680,8 +737,19 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
         //Don't change preset if in colorchange mode
         if (SaberBase::GetColorChangeMode() != SaberBase::COLOR_CHANGE_MODE_NONE) return false;
           previous_preset_fast();
+      } else {
+        // Alt blade control
+        hybrid_font.DoEffect(EFFECT_USER4, 0);
+      if (!alt_blade_on_) {
+          hybrid_font.PlayCommon(&SFX_out);
```

Comment:
WavLen and friends won't work properly if you play the sound here.
You really should override SB_Effect() and play the sound from there instead.